pkg/cmd/ports/forward.go

```diff
@@ -101,6 +102,8 @@ func ForwardPublicPort(workspaceId, projectName string, hostPort, targetPort uin
 	go func() {
 		time.Sleep(1 * time.Second)
 		views.RenderInfoMessage(fmt.Sprintf("Port available at %s", fmt.Sprintf("%s://%s.%s", *serverConfig.Frps.Protocol, subDomain, *serverConfig.Frps.Domain)))
+		var url = fmt.Sprintf("%s://%s.%s", *serverConfig.Frps.Protocol, subDomain, *serverConfig.Frps.Domain)
```

Comment:
Please move the "url" initialization one line above and use the variable in the RenderInfoMessage function

Reply:
Moved above and used in RenderInfoMessage